components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -209,16 +209,17 @@ export class SidebarChannelMenu extends React.PureComponent<Props, State> {
 
         let leaveChannel;
         if (channel.name !== Constants.DEFAULT_CHANNEL) {
-            leaveChannel =
-                    (<Menu.Group>
-                        <Menu.ItemAction
-                            id={`leave-${channel.id}`}
-                            onClick={this.handleLeaveChannel}
-                            icon={<i className='icon-close'/>}
-                            text={leaveChannelText}
-                            isDangerous={(channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) === false}
-                        />
-                    </Menu.Group>);
+            leaveChannel = (
+                <Menu.Group>
+                    <Menu.ItemAction
+                        id={`leave-${channel.id}`}
+                        onClick={this.handleLeaveChannel}
+                        icon={<i className='icon-close'/>}
```

Comment:
There is still the old implementation here with the `i` tag. we are deprecating these, so please use the icon component (`CloseIcon`) from `@mattermost/compass-icons/component`.

If you need help in implementing that fell free to reach out

Reply:
got it

Reply:
Migrated to ( CloseIcon ) from @mattermost/compass-icons/component.